tox.ini

```diff
@@ -27,6 +29,9 @@ commands =
   python --version
   coverage run test/runtests.py -v --ignore-files='(^\.|^_|^setup\.py$)' \
                                    --ignore-files=not_imported.py \
+                                   pypy: --ignore-files=_mysqldb_test.py \
+                                   pypy: --ignore-files=db_task_history_test.py \
+                                   pypy: --ignore-files=sqla_test.py \
```

Comment:
I think there's some way for nose to ignore an entire module but I might
remember wrong. Wasn't able to find it from a few quick google searches
either. Let's merge this and I can look at it later. LGTM!

On Saturday, June 20, 2015, Dave Buchfuhrer <notifications@github.com>
wrote:

> In tox.ini
> <https://github.com/spotify/luigi/pull/1014#discussion_r32884809>:
>
> > @@ -27,6 +29,9 @@ commands =
> >    python --version
> >    coverage run test/runtests.py -v --ignore-files='(^\.|^_|^setup\.py$)' \
> >                                     --ignore-files=not_imported.py \
> > +                                   pypy: --ignore-files=_mysqldb_test.py \
> > +                                   pypy: --ignore-files=db_task_history_test.py \
> > +                                   pypy: --ignore-files=sqla_test.py \
>
> @Tarrasch <https://github.com/Tarrasch> I didn't either, I just really
> hoped it would work and it did.
>
> @erikbern <https://github.com/erikbern> we can't use nose.plugins.attr
> because it'll fail with an ImportError if it tries to load these files at
> all due to not installing them. Psycopg2 doesn't work with pypy but it
> turns out that sqlalchemy and mysql-connector-python do. Updating the PR to
> remove the ignores.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/luigi/pull/1014/files#r32884809>.
>